docs(instance): add instance groups (#6783)
scaleway/docs-content: pages/instances/concepts.mdx

> An Instance is a virtual computing unit that offers resources for running applications. It functions as a self-contained entity with its own operating system, RAM, and storage. Each type of Instance has unique specifications. You can choose the type that best meets your requirements. Refer to our [Instance types documentation](/instances/reference-content/choosing-instance-type/) for further details on the various Scaleway Instance options.

Reviewer: ```suggestion
An Instance is a virtual computing unit that offers resources for running applications. It functions as a self-contained entity with its own operating system, RAM, and storage. Each type of Instance has unique specifications. You can choose the type that best meets your requirements. See the [Instance types documentation](/instances/reference-content/choosing-instance-type/) for further details on the various Scaleway Instance options.
```

Reviewer: (j'en profite)

## Instance group

> An Instance group is a collection of identical Instances, created from the same [Instance template](#instance-template) and managed together as a single resource. A group can run at a fixed size for predictable workloads, or scale automatically between a minimum and maximum Instance count based on usage metrics.<br/>

Reviewer 2: ```suggestion
An Instance group is a collection of identical Instances, created from the same [Instance template](#instance-template) and managed together as a single resource. A group can run at a fixed size for predictable workloads, or scale automatically between a minimum and maximum Instance count, based on usage metrics.<br/>
```
